### PR TITLE
Add PPP

### LINKED ISSUE #196: Draft Skeletal PPP (Melissa)

### PR DESCRIPTION
**Pull request by melissaharijanto**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -159,14 +159,14 @@ This section describes some noteworthy details on how certain features are imple
...
-The add item command will be executed by `AddItemCommand`. Items added will be stored in `InventoryList`. 
+The add item command will be executed by `AddItemCommand`. Items added will be stored in `InventoryList`.
 
 Given below is an example usage scenario and how the add item mechanism behaves at each step.
 
 Step 1. The user launches the application for the first time. The `TrackO` will be initialized with the initial TrackO state, and the `InventoryList` will contain sample data.
 
-Step 2. The user executes `addi i/keys q/10` command to add 10 keys to item list in TrackO. 
-The `addi` command creates an `AddItemCommandParser` which checks the necessary input arguments for item name (prefixed by `i/`) and quantity (prefixed by `q/`) are present before parsing the arguments into an `AddItemCommand` object. 
+Step 2. The user executes `addi i/keys q/10` command to add 10 keys to item list in TrackO.
```
Comment (janelleljt) [inline docs/DeveloperGuide.md]: Maybe can change keys to `keys`?

**Review by janelleljt — APPROVED** [review]

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (donljh) [inline docs/DeveloperGuide.md]: This is a little inconsistent from the command format, as it's missing all the compulsory fields like (`d/`, `cp/`, `sp/`, etc). Might be good to add it in

**Review by donljh — APPROVED** [review]
Added a minor comment, mergeable after your fix!

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (melissaharijanto) [reply docs/DeveloperGuide.md]: thanks for pointing that out! i'll fix it

Comment (melissaharijanto) [reply docs/DeveloperGuide.md]: yep, will do! thanks for pointing it out

**Pull request merged**